packages/editor/src/pages/Editor/store/header/selectors.ts

```diff
@@ -80,20 +93,49 @@ const getRunButton = createSelector(
           key: 'run',
           text: 'Run',
           iconProps: { iconName: 'Play' },
-          actionCreator: isTrusted
-            ? actions.editor.navigateToRun
-            : () =>
-                actions.messageBar.show({
-                  style: MessageBarType.error,
-                  text: 'You must trust the snippet before you can run it.',
-                  button: {
-                    text: 'Trust',
-                    action: actions.solutions.updateOptions({
-                      solution,
-                      options: { isUntrusted: false },
-                    }),
-                  },
-                }),
+          ...(!!queryString.parse(window.location.search).commands
```

Comment:
How about first, in line 77, doing

```
// If the URL has "commands=1" on it (from the URL specified in the manifest),
//     it means that it's run in an Office host that supports ribbon commands.
//     For such hosts, we want to split out the "run" button to alert them that
//     a better experience is to run in side-by-side mode.
const splitRunButton = queryString.parse(window.location.search)["commands"];
// 
```

And then using this variable here.

Much more readable, eh?